out/production/task3/in.txt

```diff
@@ -0,0 +1,6 @@
+5

```

Comment:
here you need to create 11 lines (one - number of objects and objects themselves)